sentry-android-core/src/test/java/io/sentry/android/core/AppStartStateTest.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.android.core
+
+import java.util.Date
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertNull
+import kotlin.test.assertSame
+
+class AppStartStateTest {
+
+    @BeforeTest
+    fun `reset instance`() {
```

Comment:
`l`: Maybe also do a reset in `@AfterTest` to leave the static `AppStartState` in a clean state.

Reply:
it'd not harm but it'd not have any effect either, as it cleans the state before each test, keeping it as it is.